packages/block-library/src/image/edit.js

```diff
@@ -497,188 +515,196 @@ export class ImageEdit extends Component {
 			</>
 		);
 
+		const AlignmentWrapper = needsAlignmentWrapper ? Block.div : Fragment;
+		const BlockContentWrapper = needsAlignmentWrapper
+			? 'figure'
+			: Block.figure;
 		// Disable reason: Each block can be selected by clicking on it
 		/* eslint-disable jsx-a11y/click-events-have-key-events */
 		return (
 			<>
 				{ controls }
-				<figure className={ classes }>
-					<ImageSize src={ url } dirtynessTrigger={ align }>
```

Comment:
Would be cool if we cool rewrite this as a hook. Something to explore separately.

Reply:
Definitely, I actually did something like that a long time ago when I was playing with React hooks before their release https://github.com/WordPress/gutenberg/pull/11161

Reply:
Awesome!